cms/static/cms/js/modules/cms.structureboard.js

```diff
@@ -112,6 +112,12 @@ $(document).ready(function () {
 				}
 			});
 
+			//fix issue #4407
+			$('.cms-toolbar-expanded .cms_dragarea .cms_draggables .cms_draggable').bind('cms.structureboard.menustate.change', function(e){
+				var drag    = $(this);
```

Comment:
why the extra space after the variable declaration?
rather use ``.on`` than ``.bind``